### PR TITLE
[#276] API support to AtomDB new queries

### LINKED ISSUE #276: Review DAS Query Engine to support new AtomDB indexes

### PR DESCRIPTION
**Pull request by eddiebrissow**
closes #276 

Added support to atom-db queries

- Fixed typos
- Added new methods to public API:
  - get_atoms_by_field
  - get_node_by_name_starting_with
  - get_atoms_by_text_field
- Changed public API methods:
  - custom_query
  - create_field_index

### CODE REVIEW

File: hyperon_das/das.py [hunk 1]
```diff
@@ -28,36 +28,46 @@ def __init__(self, system_parameters: Dict[str, Any] = {}, **kwargs) -> None:
         remote instances can be configured using kwargs options.
 
 
-        Args:
-            system_parameters (Dict[str, Any], optional): Sets the system parameters.
-                Defaults to {'running_on_server': False, 'cache_enabled': False, 'attention_broker_hostname': 'localhost', 'attention_broker_port': 27000}.
+        Args: system_parameters (Dict[str, Any], optional): Sets the system parameters. Defaults to {
+        'running_on_server': False, 'cache_enabled': False, 'attention_broker_hostname': 'localhost',
+        'attention_broker_port': 27000}.
```
Comment (angeloprobst) [inline hyperon_das/das.py:33]: Why this change?

**Review by andre-senna — APPROVED** [review]

**Review by angeloprobst — APPROVED** [review]

File: hyperon_das/das.py (hunk 1 above)
Comment (eddiebrissow) [reply hyperon_das/das.py]: merge issue, fixed
